README.md

```diff
@@ -1,5 +1,5 @@
 How to perform a SQL Injection Attack
-
+add a test line just for learning reasons
```

Comment:
Thanks for the input. Can you expand on this? Are you suggesting adding a comment that this is just for learning purposes and not for performing actual attacks?